cypress/components/carousel/carousel.cy.tsx

```diff
@@ -167,7 +169,6 @@ context("Testing Carousel component", () => {
       nextArrowButton()
```

Comment:
I know it is not a part o the change but as fas I can see the description is when `previous` arrow clicked, but in test we are still using `nextArrowButton()`